### ISSUE TITLE
Determine const visibility

### ISSUE THREAD

**Issue by rentalhost** [kalessil/phpinspectionsea#501]
It should works like methods, currently.

> PHP | Php Inspections (EA Extended) | Code Style | **Access modifiers shall be defined**

```php
// Warning: "TEST" should be declared with access modifier. 
// Quick-fix: Declare public.
const TEST = true;
```

**Comment by rentalhost** [comment]
WIP

**Comment by rentalhost** [comment]
**Closing:** PR #502

**Comment by donquixote** [comment]
It is nice to see this being supported now.

However, I think there should be a checkbox to disable this inspection for const.

I created a follow-up here: #699
